### PR TITLE
skip annotation test on windows

### PR DESCRIPTION
**Pull request by alanmcruickshank**
This test fails on windows because of path issues. Given our automated tests are all on linux, and so is the environment that this is most likely to be called, I think this is fine to skip.

### CODE REVIEW

File: test/cli/commands_test.py [hunk 1]
```diff
@@ -522,6 +523,7 @@ def test__cli__command_lint_serialize_multiple_files(serialize):
         raise Exception
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Paths won't match test on windows")
```
Comment (barrywhart) [inline test/cli/commands_test.py:526]: Looks fine. Would it be possible (and worthwhile) to modify the test so it works on Windows as well?

**Review by barrywhart — APPROVED** [review]

File: test/cli/commands_test.py (hunk 1 above)
Comment (barrywhart) [reply test/cli/commands_test.py:526]: Heh, I think I wrote this test. I'll take a quick look to see about Windows compatibility.

**Comment by barrywhart** [comment]
@alanmcruickshank: See PR #1223. With this change, the test should pass on any platform. Sorry about the broken test!

File: test/cli/commands_test.py (hunk 1 above)
Comment (alanmcruickshank) [reply test/cli/commands_test.py:526]: Don't stress - I think it's a great test for unix style platforms - which are also the main targets for it, especially given the github action. I'm fine just skipping it for windows unless we fancy working out a way of properly normalising both sides of the comparison.

**Pull request closed without merging**